CUWesternBlister/CommunityReviews
# Product table update

This pull request adds the additional products and brands to the product and brand table with the correct product/brand/category associations and all of the most recent products/brands from Luke. These .csv files are to act as the base for BCR going forward where more could get added in the future. 

Testing:

1. Make sure you have the code for the plugin pulled from this branch
2. Check your database and see that there the old product table is there. You can do this by querying for a row in the wp_bcr_products table where categoryID=3. The old table should return no rows.
3. Deactivate the plugin on your local wordpress site and make sure the products and brands tables in your database are still there before re-activating.
4. Activate the plugin and ensure that the product table exists. Query for a row with productName="Crowe Jacket" and you should get one row back with the categoryID=3 and brandID=8.
5. Check to make sure that the brands table updated as well. Query for a row with brandName="Dalbello" and you should get one row back.
